Bulldozer/Utility/AddMethods.cs

```diff
@@ -1848,6 +1848,79 @@ public static int AddBenevolenceType( RockContext rockContext, string typeName )
             return campusId;
         }
 
+        /// <summary>
+        /// Gets the campus identifier.
+        /// </summary>
+        /// <param name="property">Name of the property.</param>
+        /// <param name="includeCampusName">if set to <c>true</c> [include campus name].</param>
+        /// <returns></returns>
+        public static int? GetCampus( string campusIdString, string importPrefix, bool existingCampusesOnly = false, string possibleCampusName = null, bool addNew = false )
+        {
+            var rockContext = new RockContext();
+            int? returnCampusId = null;
+            Campus campus = null;
```

Comment:
Is this necessary to be declared outside of the if statements? Looks like it is only used in one